optashift-employee-rostering-webapp/pom.xml

```diff
@@ -28,135 +30,159 @@
+      <groupId>org.jboss.arquillian.core</groupId>
+      <artifactId>arquillian-core-api</artifactId>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>org.wildfly.arquillian</groupId>
       <artifactId>wildfly-arquillian-container-managed</artifactId>
       <scope>test</scope>
     </dependency>
 
     <dependency>
+      <groupId>org.jboss.remoting</groupId>
+      <artifactId>jboss-remoting</artifactId>
+      <version>${arquillian.version.org.jboss.remoting}</version>
+      <scope>test</scope>
+    </dependency>
+    
+   <dependency>
       <groupId>org.jboss.shrinkwrap.resolver</groupId>
       <artifactId>shrinkwrap-resolver-depchain</artifactId>
       <type>pom</type>
       <scope>test</scope>
     </dependency>
-
-
+    
     <dependency>
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
-
+    
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
-      <artifactId>resteasy-client</artifactId>
+      <artifactId>resteasy-jaxb-provider</artifactId>
       <version>${arquillian.version.resteasy}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
-      <artifactId>resteasy-jaxrs</artifactId>
+      <artifactId>resteasy-client</artifactId>
       <version>${arquillian.version.resteasy}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
-      <artifactId>resteasy-jaxb-provider</artifactId>
+      <artifactId>resteasy-jaxrs</artifactId>
       <version>${arquillian.version.resteasy}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
       <artifactId>resteasy-jackson2-provider</artifactId>
       <version>${arquillian.version.resteasy.jackson2-provider}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-core</artifactId>
       <version>${arquillian.version.jackson}</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
       <version>${arquillian.version.jackson}</version>
       <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-jaxb-annotations</artifactId>
+      <version>${arquillian.version.jackson}</version>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>com.fasterxml.jackson.jaxrs</groupId>
       <artifactId>jackson-jaxrs-base</artifactId>
       <version>${arquillian.version.jackson-jaxrs}</version>
       <scope>test</scope>
     </dependency>
-
+    
+   <dependency>
+     <groupId>dom4j</groupId>
+     <artifactId>dom4j</artifactId>
```

Comment:
Do we really need this one? If it's a workaround, add a comment when the workaround can be removed.

Reply:
Work around for hibernate (https://stackoverflow.com/questions/35533847/wildfly-10-java-lang-classcastexception-org-dom4j-documentfactory-cannot-be-ca). I tried the accepted answer, still got the exception, so I went to the second answer.